logic/round.js

```diff
@@ -107,7 +110,9 @@ Round.prototype.mergeBlockGenerator = function() {
 
 /**
  * If outsiders content, calls sql updateMissedBlocks.
- * @return {}
+ *
+ * @returns {}
```

Comment:
What does this mean? Elsewhere we either put something inside the braces or leave out the tag entirely (maybe with a todo to add it later), don't we?

Reply:
I don't know, it was already documented like this. I added a `@todo` that it should be described better. But maybe it is better to remove it completely and to add a new todo, I will do this with the next commit :)